app/components/Home/Home.js

```diff
@@ -106,9 +107,9 @@ export default class Home extends Component<Props> {
                   marginTop: 15,
                   textAlign: 'center'
                 }}
-                title="Try out the new v1.13.2"
+                title={`Welcome ${SuggestMCVersion} to GDLauncher`}
               >
-                V1.13.2 has just been released. Wanna try it out?
+                V1.15.1 has just been released. Wanna try it out?
```

Comment:
Welcome, 1.15.1, to GDLauncher?

Reply:
we can use `suggestedMCVersion` instead of hardcoding it

Reply:
Oops, forgot to change the rest of the text back to try out the new ${suggestMCVersion}.
I had copied it from welcoming the user since at that time i didn't know it was back quotes that wrapped the text instead of single quotes since they looked so similar.